spec/controllers/collection_participants_controller_spec.rb

```diff
@@ -415,7 +422,9 @@
 
         it "doesn't approve the member, displays an error and redirects" do
           get :add, params
-          it_redirects_to_with_error(collection_participants_path(collection), "#{users.last.default_pseud.name} is currently banned and cannot participate in challenges.")
+          it_redirects_to_with_error(collection_participants_path(collection), \
```

Comment:
The \ on this line shouldn't be necessary (but the next one on line 426 is necessary)